platform/posix/transport/utest/plaintext_utest.c

```diff
@@ -55,8 +55,15 @@
 /* The error returned from #send or #recv. */
 #define SEND_RECV_ERROR      -1
 
-static ServerInfo_t serverInfo;
-static NetworkContext_t networkContext;
+/* Each compilation unit must define the NetworkContext struct. */
+struct NetworkContext
+{
+    PlaintextParams_t * pParams;
+};
+
+static ServerInfo_t serverInfo = { 0 };
+static NetworkContext_t networkContext = { 0 };
```

Comment:
Don't know if you know this, but the `{ 0 }` isn't necessary since `static` is zero initialized anyway, but I guess it doesn't hurt readability to be explicit.

Reply:
You're right, but it doesn't hurt to be explicit.

Reply:
This initialization is not C89 either, so not sure if we should use it, given that we having warnings with bool.

Reply:
I recall that tests do not have to adhere to C89. These initializers are used several times in the integration tests too.
Also, is the initialization really against C89?
https://stackoverflow.com/questions/63355760/how-standard-is-the-0-initializer-in-c89